src/node_options.cc

```diff
@@ -0,0 +1,218 @@
+#include "node_options-inl.h"
+
+namespace node {
+
+DebugOptionsParser::DebugOptionsParser() {
```

Comment:
AFAIR this could be merged with `EnvironmentOptionsParser`. It was encapsulated as in preparation to something like this PR.

Reply:
Yes, this could be merged. As you are saying, it helps with encapsulation, so I don’t mind it either way.

Reply:
I would add a comment stating this, just so that future generations won't assume it's something special.

Reply:
Sounds good, done! (3c9b03a8ae10)